### PR TITLE
add pythonPackages.root_numpy, bump python2Packages.rootpy

### PR DESCRIPTION
**Pull request by veprbl**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by Mic92** [comment]
I get a test failure for `python36Packages.root_numpy`

```
======================================================================
ERROR: root_numpy (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: root_numpy
Traceback (most recent call last):
  File "/nix/store/2lb6nn8ivk1alhckv43n7734lqwbw7h9-python3-3.6.2/lib/python3.6/unittest/loader.py", line 462, in _find_test_path
    package = self._get_module_from_name(name)
  File "/nix/store/2lb6nn8ivk1alhckv43n7734lqwbw7h9-python3-3.6.2/lib/python3.6/unittest/loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "/tmp/nix-build-python3.6-root_numpy-4.7.3.drv-0/root_numpy-4.7.3/root_numpy/__init__.py", line 21, in <module>
    ROOT_VERSION = root_version_active()
  File "/tmp/nix-build-python3.6-root_numpy-4.7.3.drv-0/root_numpy-4.7.3/root_numpy/setup_utils.py", line 111, in root_version_active
    import ROOT
  File "/nix/store/j2bzxvy45331h8319dxs5ziyhrkq8i2w-root-6.10.08/lib/ROOT.py", line 24, in <module>
    import cppyy
  File "/nix/store/j2bzxvy45331h8319dxs5ziyhrkq8i2w-root-6.10.08/lib/cppyy.py", line 61, in <module>
    import libPyROOT as _backend
ImportError: dynamic module does not define module export function (PyInit_libPyROOT)
----------------------------------------------------------------------
Ran 1 test in 0.000s
FAILED (errors=1)
```

File: pkgs/development/python-modules/rootpy/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    sha256 = "0zp2bh87l3f0shiqslbvfmavfdj8m80y8fsrz8rsi5pzqj7zr1bx";
+  };
+
+  disabled = isPy3k;
+
+  propagatedBuildInputs = [ matplotlib numpy root root_numpy tables ];
+
+  meta = with lib; {
+    homepage = http://www.rootpy.org;
+    license = licenses.gpl3;
```
Comment (vyp) [inline pkgs/development/python-modules/rootpy/default.nix]: Where did you get this? https://github.com/rootpy/rootpy/blob/1.0.1/LICENSE is bsd3.

Comment (veprbl) [reply pkgs/development/python-modules/rootpy/default.nix]: @vyp They've changed license. Thanks!

**Comment by veprbl** [comment]
Thanks guys!

**Pull request merged**